src/test/scala/uk/gov/nationalarchives/notifications/LambdaSpecUtils.scala

```diff
@@ -82,4 +93,44 @@ class LambdaSpecUtils extends AnyFlatSpec with Matchers with BeforeAndAfterAll w
 
     super.afterAll()
   }
+
+  case class QueueHelper(queueUrl: String) {
+    val sqsClient: SqsClient = SqsClient.builder()
+      .region(Region.EU_WEST_2)
+      .endpointOverride(URI.create("http://localhost:8002"))
+      .build()
+
+    def send(body: String): SendMessageResponse = sqsClient.sendMessage(SendMessageRequest
+      .builder.messageBody(body).queueUrl(queueUrl).build())
+
+    def receive: List[Message] = sqsClient.receiveMessage(ReceiveMessageRequest
+      .builder
+      .maxNumberOfMessages(10)
+      .queueUrl(queueUrl)
+      .build).messages.asScala.toList
+
+    val visibilityTimeoutAttributes = new util.HashMap[QueueAttributeName, String]()
+    visibilityTimeoutAttributes.put(QueueAttributeName.VISIBILITY_TIMEOUT, (12 * 60 * 60).toString)
+
+    def createQueue: CreateQueueResponse = sqsClient.createQueue(
+      CreateQueueRequest.builder.queueName(queueUrl.split("/")(4)).attributes(visibilityTimeoutAttributes).build()
+    )
+    def deleteQueue(): DeleteQueueResponse = sqsClient.deleteQueue(DeleteQueueRequest.builder.queueUrl(queueUrl).build())
```

Comment:
Do you need the brackets on the method here? Intellij is complaining because it's defined as `deleteQueue()` and you're calling it as `deleteQueue` I quess `def deleteQueue = {` is fine to match the createQueue one do you think?